src/Sulu/Bundle/AdminBundle/Resources/js/containers/List/adapters/TableAdapter.js

```diff
@@ -6,16 +6,22 @@ import Pagination from '../../../components/Pagination';
 import Table from '../../../components/Table';
 import PaginatedLoadingStrategy from '../loadingStrategies/PaginatedLoadingStrategy';
 import FlatStructureStrategy from '../structureStrategies/FlatStructureStrategy';
+import type {LoadingStrategyInterface} from '../types';
+import FullLoadingStrategy from '../loadingStrategies/FullLoadingStrategy';
 import AbstractTableAdapter from './AbstractTableAdapter';
 
 @observer
 class TableAdapter extends AbstractTableAdapter {
-    static LoadingStrategy = PaginatedLoadingStrategy;
-
     static StructureStrategy = FlatStructureStrategy;
 
+    static paginatable = true;
```

Comment:
not needed anymore when passinge `paginated` to the constructor of the `LoadingStrategy`